Packs/Cofense-Intelligence/Integrations/CofenseIntelligenceV2/CofenseIntelligenceV2.py

```diff
@@ -0,0 +1,390 @@
+
+    # Call the Client function and get the raw response
+    result = client.threat_search_call(email=email)
+    threats = result.get('data', {}).get('threats', [])
+
+    md_data, dbot_score = threats_analysis(threats, indicator=email, threshold=params.get('email_threshold'))
+
+    dbot_score_obj = Common.DBotScore(indicator=email, indicator_type=DBotScoreType.EMAIL,
+                                      integration_name=INTEGRATION_NAME, score=dbot_score,
+                                      reliability=params.get(RELIABILITY))
+
+    email_indicator = Common.EMAIL(address=email, dbot_score=dbot_score_obj, domain=email.split('@')[1])
+    return CommandResults(
+        outputs_prefix=OUTPUT_PREFIX,
+        outputs=threats,
+        outputs_key_field='id',
+        raw_response=result,
+        readable_output=tableToMarkdown(name=f'Cofense email Reputation for email {email}', t=md_data,
+                                        headers=['Threat ID', 'Threat Types', 'Verdict', 'Executive Summary',
+                                                 'Campaign', 'Last Published']),
+        indicator=email_indicator,
+    )
+
+
+def check_md5_command(client: Client, args: Dict[str, Any], params) -> CommandResults:
+    file = args.get('file', None)
+    if not file:
+        raise ValueError('file not specified')
+    # Call the Client function and get the raw response
+    result = client.threat_search_call(file=file)
+    threats = result.get('data', {}).get('threats', [])
+    threshold = params.get('file_threshold')
+    threshold_score = SEVERITY_SCORE.get(threshold)
+    if not threshold_score:
+        raise Exception(
+            f'Cofense error: Invalid threshold value: {threshold}. Valid values are: None, Minor, Moderate or Major')
+
+    md_data = []
+    dbot_score = adjusted_score = 0
+
+    file_indicator = Common.File(md5=file, dbot_score=None)
+    for threat in threats:
+        severity_level = 0
+        block_set = threat.get('blockSet')
+        for block in block_set:
+            if block.get('impact'):
+                threat_score: int = SEVERITY_SCORE.get(block.get('impact'), 0)
+                adjusted_score = 3 if threshold_score <= threat_score else threat_score
+            severity_level = max(severity_level, adjusted_score)
+
+        for es in threat.get('executableSet'):
+            if es.get('md5Hex') == file:
+                file_indicator.sha512 = es.get('sha512Hex')
+                file_indicator.sha1 = es.get('sha1Hex')
+                file_indicator.sha256 = es.get('sha256Hex')
+                file_indicator.name = es.get('fileName')
+                file_indicator.malware_family = es.get('malwareFamily', {}).get('familyName')
+                file_indicator.extension = es.get('fileNameExtension')
+                break
+
+        threat_md_row = create_threat_md_row(threat, severity_level)
+        md_data.append(threat_md_row)
+        dbot_score = max(dbot_score, severity_level)
```

Comment:
please separate this to a different method

Reply:
Done